File: DAWES/Primer Trimestre/Actividad evaluable 1/clases/Usuario.php
<?php

class Usuario {

    private $nombre;
    private $correo;
    private $password;

    public function __construct($nombre, $correo, $password)
    {
        $this->nombre = $nombre;
        $this->correo = $correo;
        $this->password = $password;
    
    }

    public function getNombre(){
        return $this->nombre;
    }

    public function getCorreo(){
        return $this->correo;
    }

    public function getPassword(){
        return $this->password;
    }
}

File: DAWES/Primer Trimestre/Actividad 1 - Evaluable/Actividad-1-Evaluable/src/main/java/com/example/demo/clases/Usuario.java
package com.example.demo.clases;

import java.util.ArrayList;
import java.util.List;

public class Usuario {

    private String nombre, email, password;
    private List<Post> posts;

    public Usuario(String nombre, String email, String password) {
        this.nombre = nombre;
        this.email = email;
        this.password = password;
        this.posts = new ArrayList<>();
    }

    public String getNombre() {
        return nombre;
    }

    public String getEmail() {
        return email;
    }

    public String getPassword() {
        return password;
    }

    public List<Post> getPosts() {
        return posts;
    }

    public void addPost(Post post){
        this.posts.add(post);
    }
}
